backend/core/settings/prod.py

```
@@ -0,0 +1,64 @@
+from django.core.exceptions import ImproperlyConfigured
+
+from .base import *  # noqa: F401,F403
+
+
+DEBUG = False
+
+
+def _resolve_secret_key() -> str:
+    """Ensure a cryptographically strong SECRET_KEY is available."""
+
+    if SECRET_KEY != "change-me":
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)